### ISSUE TITLE
light theme is kind of lackluster

### ISSUE THREAD

**Issue by randysim** [stuysu/epsilon#125]
outlines become very faint and everything just looks white.

hard to use the site in bright theme.

though this isn't really the biggest issue because who uses light theme anyways?

**Comment by SnowyNate** [comment]
I always use the light theme for this site. We could fix this by using a gradient color as the background.

**Comment by SnowyNate** [comment]
It turns out this is much more tedious than I thought. We would have to switch from ThemeProvider to CSS, which from my research actually improves performance. Thoughts? @randysim @Rahuldeb5 @TheEgghead27

**Comment by randysim** [comment]
Ideally the site would have a minimal, sleek design, which I think having the ENTIRE background be a gradient goes against.

The problem with light theme is the lack of contrast between UI components and the background (everything just turns white), not that the background isn't interesting enough.

**Comment by TheEgghead27** [comment]
If I am understanding you correctly, you want to switch to raw CSS?

MUI ThemeProvider has ample dark/light features that work well, I wonder if the issue here is improper integration with it?

**Comment by SnowyNate** [comment]
> Ideally the site would have a minimal, sleek design, which I think having the ENTIRE background be a gradient goes against.
> 
> The problem with light theme is the lack of contrast between UI components and the background (everything just turns white), not that the background isn't interesting enough.

I see, then we would have to change the colors of the tiles themselves.

**Comment by SnowyNate** [comment]
> If I am understanding you correctly, you want to switch to raw CSS?
> 
> MUI ThemeProvider has ample dark/light features that work well, I wonder if the issue here is improper integration with it?

Yes, and use the Typescript to activate triggers in the CSS rather than handling the actual colors. Some backgrounds, such as the gradients I suggested, only work through CSS with our site. I tried using lineargradient inline and it broke the entire site.

**Comment by TheEgghead27** [comment]
I have a feeling that what we actually need (readable light mode) is very much attainable without too many CSS hacks, and while gradiants look nice, I am of the opinion this disproportionately large struggle is not necessary.

**Comment by SnowyNate** [comment]
After performing some more experimentation, changing the colors of components themselves such as the tiles is also not very sufficient. Changing the tiles to a different color other than white makes them weirdly stand out, and making them darker also looks weird. Plus, if I stick with that idea, then I have to go hunting for every component and tile that the site has. At this point, I'm going to consider my two options and go with whatever's easier.

**Comment by SnowyNate** [comment]
I have not been able to find a suitable solution for this yet, so I will unassign myself from this issue in case someone else wants to claim it.

**Comment by TheEgghead27** [comment]
I am under the impression that the light theme is *acceptable* at this time, and that there may be no need to change it substantially (only ideas I have _might_ be to adjust the cyan accent color to be darker) -- thoughts?

**Comment by SnowyNate** [comment]
It is _acceptable_ for now. If we changed the accent color to be darker... it just wouldn't look good imo for light mode. This issue hopefully will be fixed in a UI overhaul.
